precompiles/avs/avs.go

```diff
@@ -24,13 +26,15 @@ var f embed.FS
 // Precompile defines the precompiled contract for avs.
 type Precompile struct {
 	cmn.Precompile
-	avsKeeper avsKeeper.Keeper
+	avsKeeper      avsKeeper.Keeper
+	operatorKeeper operatorKeeper.Keeper
```

Comment:
Since there's already operatorKeeper within avsKeeper, still need to import new operatorKeeper rather than get it via avsKeeper.operatorKeeper?